tap_list_providers/parsers/digitalpour.py

```diff
@@ -0,0 +1,121 @@
+import json
+
+import datetime
+import dateutil.parser
+import requests
+
+class DigitalPourParser:
+    """Parser for DigitalPour based vendors."""
+
+    URL = 'https://mobile.digitalpour.com/DashboardServer/v4/MobileApp/MenuItems/{}/{}/Tap?ApiKey={}'
```

Comment:
put a `  # noqa` at the end of this line to silence the line length warning (two spaces before `#` required)

Reply:
... or not, apparently that isn't too long